admin_interface/templates/admin/base_site.html

```diff
@@ -10,22 +10,109 @@
+            --admin-interface-module-background-color: {{ theme.css_module_background_color }};
+            --admin-interface-module-background-selected-color: {{ theme.css_module_background_selected_color }};
+            --admin-interface-module-text-color: {{ theme.css_module_text_color }};
+            --admin-interface-module-link-color: {{ theme.css_module_link_color }};
+            --admin-interface-module-link-selected-color: {{ theme.css_module_link_selected_color }};
+            --admin-interface-module-link-hover-color: {{ theme.css_module_link_hover_color }};
+            --admin-interface-generic-link-color: {{ theme.css_generic_link_color }};
+            --admin-interface-generic-link-hover-color: {{ theme.css_generic_link_hover_color }};
+            --admin-interface-save-button-background-color: {{ theme.css_save_button_background_color }};
+            --admin-interface-save-button-background-hover-color: {{ theme.css_save_button_background_hover_color }};
+            --admin-interface-save-button-text-color: {{ theme.css_save_button_text_color }};
+            --admin-interface-delete-button-background-color: {{ theme.css_delete_button_background_color }};
+            --admin-interface-delete-button-background-hover-color: {{ theme.css_delete_button_background_hover_color }};
+            --admin-interface-delete-button-text-color: {{ theme.css_delete_button_text_color }};
+            --admin-interface-related-modal-background-color: {{ theme.related_modal_background_color }};
+            --admin-interface-related-modal-background-opacity: {{ theme.related_modal_background_opacity }};
+
+        {% if theme.css_header_background_color != theme.css_module_background_color %}
+            --admin-interface-main-border-top: 10px solid var(--admin-interface-module-background-color);
+        {% endif %}
+
+        {% if theme.logo_max_width > 0 %}
+            --admin-interface-log-max-width: min({{ theme.logo_max_width }}px, 100%);
+        {% else %}
+            --admin-interface-log-max-width: 100%;
+        {% endif %}
+
+        {% if theme.logo_max_height > 0 %}
+            --admin-interface-log-max-height: {{ theme.logo_max_height }}px;
+        {% endif %}
+
+        {% if theme.related_modal_rounded_corners %}
+            --admin-interface-related-modal-border-radius: 4px;
+        {% endif %}
+
+        {% if theme.css_module_rounded_corners %}
+            --admin-interface-module-border-radius: 4px;
+            --admin-interface-border-radius-0px: 0px;
+            --admin-interface-jsoneditor-border-radius: var(--admin-interface-module-border-radius);
+            --admin-interface-jsoneditor-overflow: hidden;
+        {% endif %}
+
+        {% if not theme.related_modal_close_button_visible %}
+            --admin-interface-related-modal-close-button-display: none;
+        {% endif %}
+        }
+
+        /* list-filter sticky */
+        {%  if  theme.list_filter_sticky  %}
```

Comment:
This is the only CSS printed directly in the html page, I would add a body class `list-filter-sticky` in `base-site.html`, then make this CSS working only under `.admin-interface.list-filter-sticky`, so also this can be moved to its own css static file.